prprocessor/__main__.py

```diff
@@ -282,13 +320,65 @@ async def update_redmine_on_issues(pull_request: Mapping, issues: Iterable[Issue
 
 @process_event_actions('pull_request', {'opened', 'ready_for_review', 'reopened', 'synchronize'})
 @process_webhook_payload
-async def on_pr_modified(*, pull_request: Mapping, **other) -> None:  # pylint: disable=unused-argument
-    await run_pull_request_check(pull_request)
+async def on_pr_modified(*, action: str, pull_request: Mapping, **_kw) -> None:
+    commits_valid_style = await run_pull_request_check(pull_request)
+
+    original_labels: set[str] = {label['name'] for label in pull_request['labels']}
```

Comment:
Right now the original labels are kept as strings since the config also has branch labels which won't be in the enum.